mllib/src/test/scala/org/apache/spark/ml/clustering/BisectingKMeansSuite.scala

```diff
@@ -97,6 +97,7 @@ class BisectingKMeansSuite
   test("fit, transform and summary") {
     val predictionColName = "bisecting_kmeans_prediction"
     val bkm = new BisectingKMeans().setK(k).setPredictionCol(predictionColName).setSeed(1)
+      .setMaxIter(2)
```

Comment:
So I'd be more comfortable having this in a separate test, 2 iterations is not a lot.